### PR TITLE
Cocoapod installation docs and other small doc improvements

### PR DESCRIPTION
**Pull request by ben-xD**
In trying to understand the project and trying to run the example apps successfully, and found some really small issues (and problems) I can resolve (and fix) along the way.

- Add documentation about `Secrets.xcconfig` file, because that is required to successfully run Publisher example app
- Remove Secrets.xcconfig file, because its gitignored and contains sensitive details. Instead, I provide `Example.Secrets.xcconfig` which is not gitignored. (similar reason to https://github.com/ably/ably-asset-tracking-cocoa/issues/114 )
- Remove `amplifyconfiguration.json` file
- ~~Add manual installation section, to allow users to use it before SPM/ cocoapods/ carthage releases~~ Cocoapods installation instructions via GitHub added instead. 
  - I also moved AWS Amplify dependencies into the Publisher Example app target, because its not being used by the SDK. Now this makes it consistent with Android SDK.
  - Note for **installation** from this PR: These instructions in the `README.md`will work once this code is committed to `main`. Until this PR is merged you must use the PR's branch, e.g. `  pod 'AblyAssetTracking/Publisher', :git => 'https://github.com/ably/ably-asset-tracking-cocoa', :branch => "gitignore-secrets-file"`

In summary though, it looks like not many changes (or installation steps) were needed to add the SDK to a users project.

Closes https://github.com/ably/ably-asset-tracking-cocoa/issues/114
Closes https://github.com/ably/ably-asset-tracking-cocoa/issues/149
Closes https://github.com/ably/ably-asset-tracking-cocoa/issues/155

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -169,10 +173,6 @@ Additionally, when you run tests using `Fastlane` you will see three new directo
 - At the beginning, we aim only to support iOS, but we need to keep in mind macOS and tvOS
 - Docs are written for both Swift and ObjC
 
-### iOS version requirements
-
-These SDKs require a minimum of iOS 12+ / iPadOS 12+
```
[the remark this answers is not in the document]
Comment (ben-xD) [reply README.md:174]: Yup, I moved it up to `Requirements` section, following a few other Swift libraries (e.g. [Alamofire](https://github.com/Alamofire/Alamofire), [Ricardo Pereira's SocketIO-Kit](https://github.com/ricardopereira/SocketIO-Kit)). I wanted to be consistent with other projects, but also thought it took too much space. I translated these 3 lines into 1 line.

File: README.md [hunk 2]
```diff
@@ -32,6 +32,18 @@ Visit the [Ably Asset Tracking](https://ably.com/documentation/asset-tracking) d
 - [Introducing Ably Asset Tracking - public beta now available](https://ably.com/blog/ably-asset-tracking-beta)
 - [Accurate Delivery Tracking with Navigation SDK + Ably Realtime Network](https://www.mapbox.com/blog/accurate-delivery-tracking)
 
+## Requirements
+
+- iOS 12.0+ / iPadOS 12.0+
+- Xcode 12.5+
```
Comment (mmiedlarz) [inline README.md:38]: Minor: Does SDK really need Xcode 12.5? I'm asking because 12.5 is the first version that requires macOS Big Sur so I'd be careful defining it. Wouldnt just 12.X be enough? 

As a side note: Xcode 12.5 (and Big Sur in general) is not yet available in GitHub actions.

Comment (ben-xD) [reply README.md:38]: Thanks for mentioning. I wasn't sure how low to go on that requirements. So I specified the version that I am using. I'm happy to change it to 12.x, but I cannot confirm it (my internet is too slow to download more versions of Xcode, they are 12GB each?).

Comment (mmiedlarz) [reply README.md:38]: For sure we could lower it to 12.4 as it's used now in GH Actions. It'd also satisfy me because it's the last Xcode which does work on macOS Catalina, so we won't force anyone to upgrade OS.

Comment (ben-xD) [reply README.md]: Cool, that's a good idea. Thanks

File: README.md [hunk 3]
```diff
@@ -32,6 +32,19 @@ Visit the [Ably Asset Tracking](https://ably.com/documentation/asset-tracking) d
...
+- Swift 5.3+
+- Cocoapods: 1.10+ (SPM support [planned](https://github.com/ably/ably-asset-tracking-cocoa/issues/148))
+
+
+## Installation
+
+### Manual installation
+This is a draft PR. This section in progress, to resolve [#149](https://github.com/ably/ably-asset-tracking-cocoa/issues/149).
```
[the remark this answers is not in the document]
Comment (ben-xD) [reply README.md:46]: Cool, that's been fixed

File: README.md [hunk 4]
```diff
@@ -32,6 +32,19 @@ Visit the [Ably Asset Tracking](https://ably.com/documentation/asset-tracking) d
 - [Introducing Ably Asset Tracking - public beta now available](https://ably.com/blog/ably-asset-tracking-beta)
 - [Accurate Delivery Tracking with Navigation SDK + Ably Realtime Network](https://www.mapbox.com/blog/accurate-delivery-tracking)
 
+## Requirements
+
+- iOS 12.0+ / iPadOS 12.0+
+- Xcode 12.5+
+- Swift 5.3+
+- Cocoapods: 1.10+ (SPM support [planned](https://github.com/ably/ably-asset-tracking-cocoa/issues/148))
+
```
[the remark this answers is not in the document]
Comment (ben-xD) [reply README.md]: Cool, that's been fixed

**Comment by ben-xD** [comment]
@QuintinWillison please approve if you don't have any additional comments

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (ben-xD) [reply README.md]: https://github.com/ably/ably-asset-tracking-cocoa/pull/153/commits/c1099065d90126a66c00d4fea804cdf664e42e04

**Review by QuintinWillison — APPROVED** [review]
